minishift-ci-index.yaml

```diff
@@ -8,6 +8,10 @@
         - ansicolor
         - workspace-cleanup
         - timestamps
+        - credentials-binding:
+            - text:
+                credential-id: 1a087ac4-4a16-4d26-99f5-197a9ec00c6e
```

Comment:
Hi @bstinsonmhk , I think checking in this type of access token in code is not recommended as these acts as password. For more please refer https://help.github.com/articles/creating-an-access-token-for-command-line-use/